src/utils.ts

```diff
@@ -62,3 +62,45 @@ export const getFirstSelection = (graphQuery, refId) => {
     return [];
   }
 };
+
+export const FDMResponseToDropdown = (edges) => {
+  const values = [];
+  const all = {};
+  const names = {};
+  map(edges, ({ node: { externalId, versions, name } }) => {
+    const collector = [];
+    map(versions, ({ version }) => {
+      collector.push({ label: `${version}`, value: version });
+    });
+    assignIn(all, { [externalId]: collector });
+    values.push({ value: externalId, label: name });
+    assignIn(names, { [externalId]: name });
+  });
+  return { all, names, dataModelOptions: sortBy(values, ['label']) };
+};
+export const reverseSortGet = (list, id) => sortBy(get(list, id)).reverse();
+const getNodeSelection = (selection) => {
+  const { selectionSet } = selection[0];
```

Comment:
we should use the lodash head function here, instead of directly accessing the first element on the array, that might throw an error.